app/src/main/java/com/babylon/wallet/android/data/repository/state/AccountsStateCache.kt

```diff
@@ -234,57 +233,47 @@ class AccountsStateCache @Inject constructor(
         result
     }
 
-    private fun Flow<MutableMap<AccountAddress, AccountCachedData>>.compileAccountAddressAssets(): Flow<List<AccountAddressWithAssets>> =
```

Comment:
I think we should better keep the return type, unless it is extremely obvious what it returns.

Reply:
it's so much easier to read the code

Reply:
Yes I did that but detekt complains

Reply:
>Yes I did that but detekt complains

What? 😮